src/core/annotation.js

```diff
@@ -752,10 +752,20 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
     }
   }
 
-  _processCheckBox() {
+  _processCheckBox(params) {
     if (!isName(this.data.fieldValue)) {
       return;
     }
+
+    var customAppearance = params.dict.get('AP');
+
+    if (customAppearance) {
+      let exportValues = customAppearance.get('D').getKeys();
+
+      this.data.exportValue = exportValues[0] === 'Off' ?
```

Comment:
I would like to have the unit tests extended so that we can be sure that this functionality does not break in the future. They can be found in `test/unit/annotation_spec.js`.

Reply:
Furthermore, this patch doesn't appear to do *anywhere near* enough validation done of the structure of this data itself; doing similar checks as in the `_processRadioButton` method just below would seem like a requirement to prevent unnecessary future bugs.